README.md

```diff
@@ -25,8 +46,43 @@ Here is a quick example tutorial on how to run the ML backend with a simple text
    label-studio-ml start my_ml_backend
    ```
    
-4. Run Label Studio connecting it to the running ML backend from the project settings page
+4. Start Label Studio and connect it to the running ML backend on the project settings page.
 
 ## Create your own ML backend
 
-Check examples in `label_studio_ml/examples` directory.
+Follow this tutorial to wrap your existing machine learning model code with the Label Studio ML SDK to use it as an ML backend with Label Studio. 
+
+Before you start, determine the following:
+1. What types of labeling does your model support? In other words, what are the expected inputs and outputs for your model? This informs the [Label Studio labeling config]() that your model supports.
+2. The [prediction format](https://labelstud.io/guide/predictions.html) returned by your ML backend server.
+
+This example tutorial outlines how to wrap a simple text classifier based on the [scikit-learn]() framework with the Label Studio ML SDK.
+
+First, create a class declaration. You can create a Label Studio-compatible ML backend server in one command by inheriting it from `LabelStudioMLBase`. 
+```python
+from label_studio_ml.model import LabelStudioMLBase
+
+class MyModel(LabelStudioMLBase):
+```
+
+Then, define loaders in the `__init__` method. 
+
+```python
+def __init__(self, **kwargs):
+    # don't forget to initialize base class...
+    super(MyModel, self).__init__(**kwargs)
+    self.model = self.load_my_model()
+```
+
+The inherited class contains fields for loading data specific to Label Studio:
```

Comment:
do we want to provide more guidance about when and why to use these? or is that going to be so model-dependent that we'll just mention it here?

also, are these the only fields available, or just the only ones really relevant?

Reply:
I've added few notes about this